install-dev/upgrade/php/ps_1770_preset_tab_enabled.php

```diff
@@ -28,9 +28,12 @@
  * Preset enabled new column in tabs to true for all (except for disabled modules)
  */
 function ps_1770_preset_tab_enabled() {
+    // Doctrine schema update is launched after this script so we need to, at least, add this field in database manually
+    $result = Db::getInstance()->execute('ALTER TABLE `' . _DB_PREFIX_ . 'tab` ADD enabled TINYINT(1) NOT NULL');
```

Comment:
This kind of sql request must be done in 1.7.7.0.sql

Reply:
That's what we usually do, but since this case is special (partial update because doctrine update happens too late) I preferred to centralize all the related modification in the same php function
But if you insist I can move it in the SQL file

Reply:
AFAIK it was decided to centralized SQL :thinking: And use php only if needed

Reply:
Ok, no problem I'll move this one